scripts.py

```diff
@@ -72,7 +77,7 @@ def data_directory(self):
     @classmethod
     def parse_command_line(cls, _db=None, cmd_args=None):
         parser = cls.arg_parser()
-        return parser.parse_args(cmd_args)
```

Comment:
When nosetests was run, `parse_args` would return with an unexpected additional argument that caused tests to fail, so I replaced it with `parse_known_args`. I'm not aware of all of the impact this change might have, but there are a few number of tests for `parse_command_line`, still unbroken, and most subclasses overwrite it.